changelog.d/pa-2116.added

```diff
@@ -0,0 +1 @@
+CLI: Added deep traces to `--dataflow-traces`
```

Comment:
I am not sure this is a official convention, but I would have written it like `DeepSemgrep: ...` to make clear this a DeepSemgrep related change. It can help tech writers decide where to put this in the release notes.